cicd/pipelines/running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines"]
+= Running workloads and Buildah as user namespaces on {pipelines-title}
+include::_attributes/common-attributes.adoc[]
+:context: running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines
+
+toc::[]
+
+Running {pipelines-shortname} as root user on a container can expose the host and container processes to other resources. You can reduce this type of exposure by using one of the following methods:
+
+* You can run workloads in a user namespace.
```

Comment:
```suggestion
* Run workloads in a user namespace.
```
- The lead-in sentence above already said `you can`. I think it's ok to use the imperative here to reduce the number of words.